python/tvm/meta_schedule/relay_integration.py

```diff
@@ -250,6 +252,7 @@ def tune_relay(
     seed: Optional[int] = None,
     module_equality: str = "structural",
     num_tuning_cores: Union[Literal["physical", "logical"], int] = "physical",
+    disabled_pass: Optional[Union[List[str], Set[str], Tuple[str]]] = None,
```

Comment:
Could you please add information about this parameter to the docstring?

Reply:
Done